### PR TITLE
Team 1 jigyasu

### PR DESCRIPTION
**Pull request by JigyasuMittal**

### CODE REVIEW

File: source/core/src/main/com/csse3200/game/components/mainmenu/MainMenuActions.java [hunk 1]
```diff
@@ -51,6 +51,7 @@ private void onLoad() {
     private void onExit() {
         logger.info("Exit game");
         game.exit();
+        System.exit(-1);
```
Comment (IanP96) [inline source/core/src/main/com/csse3200/game/components/mainmenu/MainMenuActions.java:54]: Why is the exit status -1? I'm pretty sure -1 means exited with error. Also the `game.exit()` should take care of this anyway

File: source/core/src/main/com/csse3200/game/services/ResourceService.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
 public class ResourceService implements Disposable {
 
   private static final Logger logger = LoggerFactory.getLogger(ResourceService.class);
-  private final AssetManager assetManager;
+  public final AssetManager assetManager;
```
Comment (IanP96) [inline source/core/src/main/com/csse3200/game/services/ResourceService.java:19]: why is assetManager public?

**Review by IanP96 — CHANGES_REQUESTED** [review]
Looks mostly good but please address the comments

File: source/core/src/main/com/csse3200/game/components/maingame/MainGameExitDisplay.java [hunk 3]
```diff
@@ -46,30 +55,86 @@ private void addActors() {
     iconDownSheet.getWidth() / 6,
     iconDownSheet.getHeight() / 6);
 
+    /*TextureAtlas atlas = ServiceLocator.getResourceService()
+                .getAsset("flat-earth/skin/flat-earth-ui.atlas", TextureAtlas.class);
+    TextureRegion windowTexture = atlas.findRegion("list");*/
+    Texture blackDotTrans = ServiceLocator.getResourceService()
+                .getAsset("images/black_dot_transparent.png", Texture.class);
```
[the remark this answers is not in the document]
Comment (yashwanthg22) [reply source/core/src/main/com/csse3200/game/components/maingame/MainGameExitDisplay.java:62]: Even I'm having the same issue, The images is not updated in the asset folder.

**Pull request merged**